### PR TITLE
C20-976: Copy updates and better links

### PR DESCRIPTION
**Pull request by karanjthakkar**
### Changes

- Make currency routes count for MCA copy consistent to `50+`
- Remove `Mastercard®` from the MCA copy
- Make personal business tab links locale aware

## Checklist
- [x] I have considered the impact of this change and added a [Change Classification](
https://transferwise.atlassian.net/wiki/spaces/EKB/pages/1401189673/Change+Classifications+and+Expectations) Label (`change:standard`, `change:impactful`, `change:emergency`)
- [x] Change meets or does not compromise the [Baseline Security Requirements](https://transferwise.atlassian.net/wiki/spaces/EKB/pages/434929973/Baseline+Security+Requirements)

### CODE REVIEW

File: translations/personal_nav/messages.json [hunk 1]
```diff
@@ -5,9 +5,9 @@
...
   "public-navigation.money-transfer.money-tracker": "Track your transfer",
-  "public-navigation.multi-currency-account.nocard.description": "A multi-currency account for 40+ currencies. Send, receive and convert instantly with no hidden fees.",
-  "public-navigation.multi-currency-account.card.description": "Manage your money across 50+ currencies. Send at the real exchange rate, and spend with a TransferWise debit Mastercard®.",
-  "public-navigation.multi-currency-account.free.card.description": "Manage your money across 50+ currencies. Send at the real exchange rate, and spend with a free TransferWise debit Mastercard®.",
+  "public-navigation.multi-currency-account.nocard.description": "A multi-currency account for 70+ currencies. Send, receive and convert instantly with no hidden fees.",
+  "public-navigation.multi-currency-account.card.description": "Receive money from 70+ countires. Send at the real exchange rate, and spend with a TransferWise debit card.",
+  "public-navigation.multi-currency-account.free.card.description": "Receive money from 70+ countires. Send at the real exchange rate, and spend with a free TransferWise debit card.",
```
Comment (twrdjpalmer) [inline translations/personal_nav/messages.json:10]: Ah, this is my bad, I understood the conversation yesterday as the number we can send to (which is 70+ for UK). Receive is actually more like 7, and I'm not sure what the correct number would be for the line 8.

Might be best to try and make the copy agnostic of the number, given saying one thing here and another else where is quite confusing for the customer

Comment (karanjthakkar) [reply translations/personal_nav/messages.json]: Based on Loz's suggestion, I've sent the copy to Karam to confirm. It's possible we might keep the count as before.

Comment (karanjthakkar) [reply translations/personal_nav/messages.json]: @twrdjpalmer I've updated the copy. This has now been confirmed by Karam and Felipe.

**Review by danlaush — APPROVED** [review]

**Review by twrdjpalmer — APPROVED** [review]

**Comment by karanjthakkar** [comment]
Waiting on translations to finish before merge....

**Comment by karanjthakkar** [comment]
Translations complete, hence merging.

**Pull request merged**